test-configs.yaml

```diff
@@ -744,6 +744,8 @@ device_types:
     mach: rockchip
     class: arm64-dtb
     boot_method: depthcharge
+    filters:
+       - blacklist: {kernel: ['v4.20-']}
```

Comment:
One issue with putting the filter here is that it will blacklist kernel revisions for all tests, whereas we only need to blacklist `igt` on kernels earlier than 4.20.

Reply:
Is it possible to use the filter feature for an individual test plan or is it something that needs to be implemented?

Reply:
I believe this is something that needs implemented, to add support for filters in test config objects.

Reply:
Actually it was already there :) (in TestConfig)